src/main/java/seedu/address/model/event/Event.java

```diff
@@ -59,6 +59,31 @@ public Description getDescription() {
         return description;
     }
 
+    /**
+     * Changes the given Name from {@code original} to {@code replacement} if
```

Comment:
maybe be more specific here "Change the given Friendname" since Name could refer to EventName and FriendName